docker-compose.yaml

```diff
@@ -71,8 +71,8 @@ services:
     container_name: civil-servant-registry
     environment:
       - SPRING_PROFILES_ACTIVE=default
-      - CHECK_TOKEN_URL=http://identity:8080/oauth/check_token
-    image: cshr/civil-servant-registry-service:9fe7cd23d21b7428786e14f6a8c3a548b6a7211f
```

Comment:
Should this be changed?

Reply:
Yeah, check token url is still used but derived from the oauth service url now along with a couple of other endpoints.